kalite/django_cherrypy_wsgiserver/cherrypyserver.py

```diff
@@ -135,14 +135,6 @@ def run_cherrypy_server(host="127.0.0.1", port=None, threads=None, daemonize=Fal
     if daemonize:
         if not pidfile:
             pidfile = '~/cpwsgi_%d.pid' % port
-        
-        # benjaoming: stopping the server is an explicit logic that has already
-        # been implemented other places. Killing some process related to a
-        # possibly out-dated pidfile is not exactly best practice
-        # stop_server(pidfile)
-
-        from django.utils.daemonize import become_daemon
-        become_daemon()
```

Comment:
The pidfile is necessary for `stop` and `status`. Also see #3776, where I also write the listen port to the pidfile, enabling `stop` and `start` to work when kalite is listening on a non-default port. If it's not handled here, it should be handled elsewhere. `kalitectl.py`?